security/webauthn/src/test/java/io/quarkus/ts/security/webauthn/AbstractWebAuthnTest.java

```diff
@@ -101,23 +114,35 @@ public void testRegisterWebAuthnUser() {
     @Test
     @Order(6)
     public void testRegisterSameUserName() {
-        MyWebAuthnHardware myWebAuthnHardware = new MyWebAuthnHardware();
-        String challenge = getChallenge(USERNAME, cookieFilter);
+        MyWebAuthnHardware myWebAuthnHardware = new MyWebAuthnHardware(url);
+        String challenge = getRegistrationChallenge(USERNAME, cookieFilter);
         JsonObject registrationJson = myWebAuthnHardware.makeRegistrationJson(challenge);
-        invokeCallback(registrationJson, cookieFilter);
-        verifyLoggedIn(cookieFilter, USERNAME, User.USER);
+        ExtractableResponse<Response> response = RestAssured
+                .given()
+                .queryParam("userName", USERNAME)
+                .body(registrationJson.encode())
+                .filter(cookieFilter)
+                .contentType(ContentType.JSON)
+                .log().ifValidationFails()
+                .post(REGISTER_URL, new Object[0])
```

Comment:
The second parameter, `new Object[0],` is not needed from my point of view, but we can keep it.

Reply:
This is a copy of the same code further, I can fix both.